lib/instrumentation/http-shared.js

```diff
@@ -24,8 +24,10 @@ exports.instrumentRequest = function (agent, moduleName) {
           agent._instrumentation.currentTransaction = null
         } else {
           var traceparent = req.headers['elastic-apm-traceparent'] || req.headers.traceparent
+          var tracestate = req.headers['elastic-apm-tracestate'] || req.headers.tracestate
```

Comment:
++ to dropping the `elastic-apm-tracestate`. We can reintroduce if required, but right now we don't need it.

Reply:
Good call -- no need to introduce legacy, pre-spec code unless there's a specific need to (and it will be relativly easy to add if there is).  Changed this to check `req.headers.tracestate` only.